build.cmd

```diff
@@ -49,12 +49,9 @@ set RUNNER=".\packages\Microsoft.Net.Compilers.%CSI_VERSION%\tools\csi.exe"
 
 @echo On
 
-%RUNNER% .\artifacts\files\internal\runner.csx || goto :error
-%RUNNER% .\artifacts\files\internal\target-runner.csx || goto :error
-%RUNNER% .\artifacts\files\internal\target-runner-options.csx || goto :error
-%RUNNER% .\artifacts\files\internal\util.csx || goto :error
-%RUNNER% .\artifacts\files\simple-targets-target.csx || goto :error
-%RUNNER% .\artifacts\files\simple-targets.csx || goto :error
+for /R .\artifacts\files\ %%f in (*.csx) do (
+      %RUNNER% "%%f" || goto :error

```

Comment:
Does the indent have to be this big, or can it be 2 chars, to match the rest of the file (see the `if` statements further up)?

Reply:
Odd, I thought I'd conformed. I'd blame VS Code, saying that it re-indented for me, but when I reduced to 2 spaces, it left the file alone. Something foolish on my part.